public/static/docs/command-reference/diff.md

```diff
@@ -1,28 +1,29 @@
 # diff
 
-Show changes between versions of the <abbr>DVC project</abbr>. It can be
+Show changes between revisions of the <abbr>DVC repository</abbr>. It can be
 narrowed down to specific target files and directories under DVC control.
 
-> This command requires that the project is a [Git](https://git-scm.com/)
-> repository.
+> This command requires that the <abbr>project</abbr> is a
+> [Git](https://git-scm.com/) repository.
 
 ## Synopsis
 
 ```usage
 usage: dvc diff [-h] [-q | -v] [-t TARGET] a_ref [b_ref]
 
 positional arguments:
-  a_ref                 Git reference from which diff calculates
-  b_ref                 Git reference until which diff calculates, if
-                        omitted diff shows the difference between
-                        current HEAD and a_ref
+  a_ref     Git reference from which the diff begins
```

Comment:
are we renaming these arguments in DVC? If not, let's keep reference instead revision in this doc, otherwise it's not clear what is happening. We have a mix of everything.

Reply:
We are updating them in core, yes: iterative/dvc/pull/3244

Reply:
> let's keep reference instead revision in this doc

Sorry, didn't get that. The term "reference" is not being changed here.

Reply:
OK I think you mean that the term "revision" in other places in this doc can be confusing. I changed them to "reference" as much as possible now. See 961b5135.

Reply:
so, is it correct from the DVC core perspective that we call there params refs? or should they be called revisions? How does git diff call them?

Reply:
It's technically correct but probably unnecessary. `git diff` just calls them `<commit>`s.

I think for now at least DVC core should call everything "revision" since that term is already used a lot for `dvc get/import` (including import stage DVC-files).

More in this discussion: https://github.com/iterative/dvc/pull/3244#pullrequestreview-348990111.